### ISSUE TITLE
Why attributes names are variables ?

### ISSUE THREAD

**Issue by ryuran** [nico3333fr/jquery-accessible-hide-show-aria#14]
Hello

I want rework this code and I need to know why uses variables for this attributes names. Can you help ?

https://github.com/nico3333fr/jquery-accessible-hide-show-aria/blob/master/jquery-accessible-hide-show-aria.js#L10-L13

> these are recommended settings by a11y experts. You may update to fulfill your needs, but be sure of what you’re doing.

**Comment by nico3333fr** [comment]
Hello,

this is due to the support of ARIA depending of configuration (browser/speech synthesis/etc.). For example, you can use all ARIA attributes on a NVDA/Firefox, but on on VoiceOver/Safari, it can be a real pain in the ass to use on a website.

In full application mode, ARIA attrbiutes will probably be the best choice.

So the configuration suggested by experts (only aria-expanded) is the most "sure" and "passe-partout" for a majority of websites, but for special cases for experts, they could appreciate to tweak it (you may see these vars as constants).

**Comment by ryuran** [comment]
So full aria support should be a better choice but with some browsers/screenreaders it can be too noisy for users.
This is why it is good to keep it as an option.
OK
